tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "target": "es5",
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "lib": [
+      "es6"
+    ],
+    "declaration": true,
+    "isolatedModules": true,
+    "allowSyntheticDefaultImports": true, 
+    "esModuleInterop": true,
+    "strict": true,
+  },
+  "exclude": [
+    "node_modules",
+    "android",
+    "examples",
+    "ios"
```

Comment:
add "dist", then you don't have to remove the dist directory every time you rebuild (mentioned in other comment)

Reply:
Done